src/reversion/revisions.py

```diff
@@ -141,12 +135,15 @@ def get_serialized_data(self, obj):
         return serializers.serialize(
             self.get_serialization_format(),
             (obj,),
-            fields = list(self.get_fields_to_serialize()),
+            fields=list(self.get_fields_to_serialize()),
         )
 
     def get_version_data(self, obj, db=None):
         """Creates the version data to be saved to the version model."""
         object_id = force_text(obj.pk)
+
+        from django.contrib.contenttypes.models import ContentType
```

Comment:
V 1.9 mám problém s tím, že když to není lokální import, tak to skončí AppRegistryNotReady